smoke-tests/apps/TraceLogBackUsingAgent/src/smokeTest/resources/applicationinsights.json

```diff
@@ -1,4 +1,11 @@
 {
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
```

Comment:
the reason why you add this config in every json file is not to apply litmitPerSecond sampling?